src/plugins/styled-text-tracks/styled-text-tracks.scss

```diff
@@ -0,0 +1,71 @@
+.cld-video-player {
+  // Positions
+  // Center
```

Comment:
I would say the positions are pretty clear from the class names (if you agree with me applies to the following classes as well, including themes)
```suggestion
```

Reply:
Yeah, I was using them as more of separators rather than to clarify meaning. Removed.